source/numerics/fe_field_function.inst.in

```diff
@@ -18,7 +18,10 @@
 for (DoFHandler : DOFHANDLER_TEMPLATES; VECTOR : VECTOR_TYPES;
      deal_II_dimension : DIMENSIONS)
   {
-    template class FEFieldFunction<deal_II_dimension,
-                                   DoFHandler<deal_II_dimension>,
-                                   VECTOR>;
+    namespace Functions
+    \{
```

Comment:
Better, yes. This could also have been done in a separate PR, but let's see how long it takes to get this merged.

Reply:
This is a historic remnant of this PR :)

It feels out of place now. I can move it to a separate PR.

Reply:
I tend to break these things out of my PRs because they're uncontroversial and easy to merge. But let's not make our lives unnecessarily complicated and just leave it here.